subnet/subnet_contract.go

```diff
@@ -12,15 +13,35 @@ import (
 const (
 	// ErrInvalidSubnetID is thrown when subnet id is not a slice of 4 bytes.
 	ErrInvalidSubnetID = "invalid subnet ID"
-	// ErrInvalidOwner is thrown when owner has invalid format.
-	ErrInvalidOwner = "invalid owner"
 	// ErrAlreadyExists is thrown when id already exists.
 	ErrAlreadyExists = "subnet id already exists"
-	// ErrNotExist is thrown when id doesn't exist.
-	ErrNotExist = "subnet id doesn't exist"
+	// ErrSubNotExist is thrown when id doesn't exist.
+	ErrSubNotExist = "subnet id doesn't exist"
```

Comment:
Redundant renaming since what exactly is missing is clear from the package name. Anyway `Sub` -> `Subnet`.